### DISCUSSION TITLE
You cannot use Cocoapods to pull TrustWalletCore？

### DISCUSSION THREAD

**Discussion by lianghuiboy** [WalletConnect/walletconnect-monorepo#1432] [Developer Support]
<img width="585" alt="image" src="https://user-images.githubusercontent.com/17612082/189509525-35b30f64-0463-44b9-8caa-a5eb5e1650b2.png">

**Comment by lianghuiboy** [comment]
WalletConnet 1.0 Example

**Reply by Arnelhub111** [reply]
https://github.com/WalletConnect/walletconnect-monorepo/discussions/1432#discussioncomment-3616950
